### PR TITLE
Update to v1.2.0

### PR DESCRIPTION
**Pull request by jhonatasrm**
- Updated README.md
- Updated icons (to AMO)
- Updated gif
- Updated minimum version to 62.0
- Fixed issue with select (dark theme)

### CODE REVIEW

**Comment by dannycolin** [comment]
> Updated minimum version to 62.0

FYI, It still works on 58 :)

File: src/css/components/select.css [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 select {
     -moz-appearance: none;
-    background-color: var(--button-background);
+    background-color: #4A4A4F;
```
Comment (dannycolin) [inline src/css/components/select.css]: @jhonatasrm can you create a CSS variable because using a hardcoded value will break the light theme

Comment (jhonatasrm) [reply src/css/components/select.css]: Yes... I will look at the settings.css 
🙂

**Comment by jhonatasrm** [comment]
> > Updated minimum version to 62.0
> 
> FYI, It still works on 58 :)

Was a message from AMO.. (sidebar)

File: src/css/components/select.css (hunk 1 above)
Comment (dannycolin) [reply src/css/components/select.css]: I fixed the missing values for the light theme.

**Comment by jhonatasrm** [comment]
@dannycolin we have an error "Icon must be square" when submitted in AMO, so I changed the sumo.svg to sumo.png (sumo16.png, sumo32.png, sumo64.png) at manifest.json for now. We need to take a look at that after.

**Comment by dannycolin** [comment]
> > > Updated minimum version to 62.0
> > 
> > 
> > FYI, It still works on 58 :)
> 
> Was a message from AMO.. (sidebar)

Strange. I guess we could even bump it to 68 since both 58 and 62 are End of Life. :P 

>  "Icon must be square" when submitted in AMO, so I changed the sumo.svg to sumo.png (sumo16.png, sumo32.png, sumo64.png) at manifest.json for now. We need to take a look at that after.

Ah yes. I saw the warning today but didn't had the time to look at time. Thanks for the temporary patch ;).

**Comment by jhonatasrm** [comment]
> Strange. I guess we could even bump it to 68 since both 58 and 62 are End of Life. :P

Yes... I don't see any problem, because this Add-on is for contributors, so they need the browser updated... hahahahhaha"

**Comment by WesleyBranton** [comment]
> > Updated minimum version to 62.0
>
> FYI, It still works on 58 :)

Just for clarification, it needs Firefox 62 minimum because it uses the `open_at_install` property in the manifest for the sidebar.

**Pull request merged**